File: backend/src/main/java/info/gamewise/lor/videos/LoRVideosApplication.java
package info.gamewise.lor.videos;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.scheduling.annotation.EnableScheduling;

@EnableScheduling
@SpringBootApplication
public class LoRVideosApplication {

    public static void main(String[] args) {
        SpringApplication.run(LoRVideosApplication.class, args);
    }

}

File: backend/src/main/kotlin/info/gamewise/lor/videos/LoRVideosApplication.kt
package info.gamewise.lor.videos

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.cache.annotation.EnableCaching
import org.springframework.scheduling.annotation.EnableScheduling

@EnableCaching
@EnableScheduling
@SpringBootApplication
open class LoRVideosApplication

fun main(args: Array<String>) {
    SpringApplication.run(LoRVideosApplication::class.java, *args)
}